app/src/main/java/de/hbch/traewelling/api/ApiService.kt

```diff
@@ -182,7 +181,7 @@ interface UserService {
 
     @DELETE("user/destroyMute")
     fun unmuteUser(
-        @Query("userId") userId: Int
+        @Path("userId") userId: Int
```

Comment:
Is this endpoint really changed?

Reply:
I actually don't have enough time to look into this rn, can you look at this?

Reply:
After looking at this it seems like I wanted to put that @Query annotation at the function you reviewed above